backend/src/auth/dtos/forgot-password.dto.ts

```diff
@@ -0,0 +1,7 @@
+import { IsEmail, IsString } from 'class-validator'
+
+export class ForgotPasswordDTO {
+  @IsString()
+  @IsEmail()
+  email: string
+}
```

Comment:
```suggestion
  @IsEmail()
  email: string
}
```
the description of IsEmail says "If given value is not a string, then it returns false"
so its duplicate